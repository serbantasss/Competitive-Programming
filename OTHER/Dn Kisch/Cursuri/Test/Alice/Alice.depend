# depslib dependency file v1.0
1427640450 source:c:\users\adi\desktop\test\alice\main.cpp
	<cstdio>

