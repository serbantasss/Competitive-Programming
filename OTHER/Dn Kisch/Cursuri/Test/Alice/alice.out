46
86
94